TheMET/TheMET/ViewModels/AboutMETViewModel.swift

```diff
@@ -43,8 +44,73 @@ class AboutMETViewModel {
         let aboutAppNavigationController = UINavigationController(rootViewController: aboutAppViewController)
         presentingController.present(aboutAppNavigationController, animated: true)
     }
+    
+    func topLocatonViewDidTap() {
+        self.locationViewDidTap(latitude: self.topMapViewData.latitude, longitude: self.topMapViewData.longitude, name: NSLocalizedString("about_met_the_met_fifth_avenue_title_lable", comment: ""), address: "The+Metropolitan+Museum+of+Art,+1000+Fifth+Avenue+New+York,+NY+10028")
+    }
+    
+    func bottomLocatonViewDidTap() {
+        self.locationViewDidTap(latitude: self.bottomMapViewData.latitude, longitude: self.bottomMapViewData.longitude, name: NSLocalizedString("about_met_the_met_cloisters_title_lable", comment: ""), address: "The+Met+Cloisters,+99+Margaret+Corbin+Drive+Fort+Tryon+Park+New+York,+NY+10040")
+    }
+    
+    private func locationViewDidTap(latitude: Double, longitude: Double, name: String, address: String) {
+        guard let presentingController = self.presentingControllerProvider() else {
+            return
+        }
+        if UIApplication.shared.canOpenURL(NSURL(string:"comgooglemaps://")! as URL) {
+            let alertController = UIAlertController(
+                title: nil,
+                message: nil,
+                preferredStyle: .actionSheet
+            )
+            let googleAction = UIAlertAction(
+                title: "Show in Google Maps",
+                style: .default,
+                handler: { _ in
+                    UIApplication.shared.open(
```

Comment:
Let's wrap with call to a new method `openInGoogleMap` (for consistency with `openInAppleMap`)